### DISCUSSION TITLE
beforeRouteLeave not working in Nova Tool

### DISCUSSION THREAD

**Discussion by julianlaibach** [laravel/nova-issues#3793] [Q&A] [answered]
- Laravel Version: 8
- Nova Version: 3.31.0
- PHP Version: 8.0
- Database Driver & Version: x
- Browser type and version: #.#

### Description:
I need to use the beforeRouteLeave in a nova Tool but the event never gets fired. Is there any additional step to do other than calling the function on mount in the Parent Component or is it currently not supported?

Thank you in advance!

[Vue Router Documentation](https://router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards)

**Comment by crynobone** [comment, marked as answer]
I wasn't able to replicate the issue: https://github.com/laravel/nova-dusk-suite/commit/ebaa25a0d1ea197e0da7bf154bd2f9430f09f06e

![image](https://user-images.githubusercontent.com/172966/160313632-5f80df9b-3d3a-4325-b60f-e2bf90dd5f0e.png)
